front/src/utils/showPrice.ts

```diff
@@ -0,0 +1,8 @@
+export const showPrice = (number: number): string => {
```

Comment:
number: number to nieladnie wyglada, lepiej unikac uzywania takich samych nazw jak te zarezerwowane np. przez TS

No i mozesz zrobic nizej po prostu 
return `${mainPrice},${restPrice}`

Reply:
Done.